jekyll/_cci2/workflows.md

```diff
@@ -157,6 +157,30 @@ Following is a screenshot of the Approval dialog box that appears when you click
 
 ![Approval Dialog in On Hold Workflow]({{ site.baseurl }}/assets/img/docs/approval_job_dialog.png)
 
+## Scheduling a Workflow
+
+Workflows may be configured to run on a set schedule by using the `triggers:` key with the `type: scheduled` key. The `triggers` key is **only** added under your `workflow` key. This feature enables you to configure a workflow with `type: scheduled` at a specific `frequency` by using `cron` syntax to represent Coordinated Universal Time (UTC/GMT). Jobs run on the specified branches within the hour defined in the `frequency` key (frequency is not guaranteed below the 30 minute increment). The following example shows a `commit` workflow that runs the `test` and `deploy` jobs for every commit to the repo followed by a `nightly` workflow triggered at 12:00am UTC every day of every month of every year that runs the `coverage` job on the master and beta branches:
+
+```
+workflows:
+  version: scheduled-workflows-mvp
+  commit:
+    jobs:
+      - test
+      - deploy
+  nightly:
+    triggers:
+      - type: scheduled
+        frequency: "0 0 * * *"
+        branches:
+          - master
+          - beta
+    jobs:
+      - coverage
+```
+
+It is possible to configure contexts and all additional options associated with workflows for a `triggers` run, except `filters`. Refer to the [crontab man page](http://pubs.opengroup.org/onlinepubs/7908799/xcu/crontab.html) for `cron` syntax basics. 
```

Comment:
It'd be dope to provide some motivation for using scheduled workflows here. Also, it would be nice to focus just on the scheduled workflow bits. What do you think of:

> Workflows that are resource-intensive or that generate reports can be run on a schedule rather than on every commit. This is configured by adding a scheduled trigger to the configuration of the workflow. In the example below, the `nightly` workflow is configured with a `frequency` of every day at 12:00am UTC. The `frequency` is specified using [POSIX crontab syntax](http://pubs.opengroup.org/onlinepubs/7908799/xcu/crontab.html). The workflow will be run on the `master` and `beta` branches. The `commit` workflow has no scheduled trigger configured, so it will run on the push of every commit.
> ```
> workflows:
>   version: 2
>   nightly:
>     triggers:
>       - type: scheduled
>         frequency: "0 0 * * *"
>         branches:
>           - master
>           - beta
>     jobs:
>       - coverage
>   commit:
>     jobs:
>       - test
>       - deploy
> ```
?
(The `version` key needs to be fixed too).

Reply:
thanks, I included this new text and removed some other bits, please have another look and let me know if we should keep the part here about 'no guarantee' or rework it.

Reply:
@michelle-luna awesome, looks good. But yeah, let's drop the bit about no guarantees.

Reply:
thanks, removed

Reply:
@kunal let me know what you think, this is what I had before "Jobs run on the specified branches within the hour defined in the `frequency` key (frequency is not guaranteed below the 30 minute increment)." and I removed it per the response from Nate above.